src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getArea() {
+        return radius * radius * Math.PI;
+    }
```

Comment:
Consider declaring 'Math.PI' as a constant field within the class for better code readability. While 'Math.PI' is a well-known constant, defining it as a static final field in your class with a meaningful name would adhere to best practices for magic numbers.